src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewModel.swift

```diff
@@ -20,10 +20,11 @@ struct DeltaOnboardingNewVersionFeaturesViewModel {
 			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature115SwitzerlandTitle, description: AppStrings.NewVersionFeatures.feature115SwitzerlandDescription)
 		)
 		
+		// This feature has removed from 1.15 but will arrive later
 		// Error analytics log
-		self.newVersionFeatures.append(
-			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature115ErrorAnalyticsLogTitle, description: AppStrings.NewVersionFeatures.feature115ErrorAnalyticsLogDescription)
-		)
+		//		self.newVersionFeatures.append(
+		//			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature115ErrorAnalyticsLogTitle, description: AppStrings.NewVersionFeatures.feature115ErrorAnalyticsLogDescription)
+		//		)
 	}
```

Comment:
why not delete it?

Reply:
Because it might come in the next 1.15 RC